wrappers/core18.go

```diff
@@ -204,5 +204,71 @@ func writeSnapdServicesOnCore(s *snap.Info, inter interacter) error {
 		return err
 	}
 
+	// Handle the user services
+	if err := writeSnapdUserServicesOnCore(s, inter); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func writeSnapdUserServicesOnCore(s *snap.Info, inter interacter) error {
+	// Ensure /etc/systemd/user exists
+	if err := os.MkdirAll(dirs.SnapUserServicesDir, 0755); err != nil {
```

Comment:
What is the configuration of this directory with regards to writable paths?

Reply:
In `/snap/core18/current/etc/system-image/writable-paths` we have:

    # systemd
    /etc/systemd                            auto                    persistent  transition  none

So it is covered the same way `/etc/systemd/system` is.